### PR TITLE
Added ford annotation for DataFlagEncoder

### PR DESCRIPTION
**Pull request by boldtrn**
As discussed in #236 fords are not routable with the car profile.

The DataFlagEncoder already allows to route through fords. This PR adds annotations to every ford, notifying users that the route contains a ford. The notification is only seen when looking at the instructions, no actual warning is produced.

Right now every Ford is routable. The question is if we actually want that. In #236 there is a discussion about a couple of attributes, including `depth, seasonal, intermittent`, maybe even more :). These checks are not done by the DataFlagEncoder or the GenericWeighting. Do we want to add that?

TODOS:

- [x] Translate the key `way_contains_ford` or decide on a different key.

### CODE REVIEW

File: core/src/test/java/com/graphhopper/routing/util/DataFlagEncoderTest.java [hunk 1]
```diff
@@ -378,4 +385,28 @@ public void testSpatialId() {
         assertEquals(5, encoder.getMaxspeed(e3, -1, false), .1);
         assertEquals(-1, encoder.getMaxspeed(e4, -1, false), .1);
     }
+
+    private Translation getTranslation(final String msg) {
```
Comment (karussell) [inline core/src/test/java/com/graphhopper/routing/util/DataFlagEncoderTest.java]: Will TranslationMapTest.SINGLETON work here?

Comment (boldtrn) [inline core/src/test/java/com/graphhopper/routing/util/DataFlagEncoderTest.java]: Yes, thanks.

**Comment by boldtrn** [comment]
ford_gh

I updated the PR with instructions and updated the Test accordingly.

File: core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java [hunk 2]
```diff
@@ -413,7 +412,7 @@ public long handleWayTags(ReaderWay way, long allowed, long relationFlags) {
             flags = accessEncoder.setValue(flags, getAccessValue(way));
 
 
-            if(spatialRuleLookupEnabled()){
+            if (spatialRuleLookupEnabled()) {
```
Comment (karussell) [inline core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java]: I've seen this in some other PR but forgot to comment: if we make this method public we have to follow java coding rules: `isSpatialRuleLookupEnabled`

**Comment by karussell** [comment]
BTW: with DataFlagEncoder.getTransportModeMap we can even enable/disable fords per request

**Pull request merged**